frontend/src/components/pages/LogRecords.tsx

```diff
@@ -45,17 +47,21 @@ const mockRecords = [
     Date: "Jan 21",
     Time: "3:20 AM",
     Resident: "MB404",
-    Note: "During security check, MB404 was making some noise. TSW warned her to be quiet. She yelled on TSW behind the door, yelled, and swore (f..uck of......). TSW told her that I will call the police if she continues. Then she came down for laundry.",
+    Note:
+      "During security check, MB404 was making some noise. TSW warned her to be quiet. She yelled on TSW behind the door, yelled, and swore (f..uck of......). TSW told her that I will call the police if she continues. Then she came down for laundry.",
     Employee: "Huseyin",
     Attn_To: "John Doe",
   },
 ];
 
-const LogRecords = (): React.ReactElement => {
```

Comment:
Any reason for removing this?

Reply:
not really, I'll add it back since its good practice to indicate this is a React element